### PR TITLE
[nearlib dev] recreate dev account if lookup fails

### PR DESCRIPTION
**Pull request by janedegtiareva**
(e.g. we had to delete db)

### CODE REVIEW

File: nearlib/dev.js [hunk 1]
```diff
@@ -9,16 +9,23 @@ module.exports = {
...
         let tempUserAccountId = window.localStorage.getItem(localStorageAccountIdKey);
         if (tempUserAccountId) {
-            return tempUserAccountId;
+            // Make sure the user actually exists and recreate it if it doesn't
+            const accountLib = new nearlib.Account(near.nearClient);
+            try {
+                const accountLookupResult = await accountLib.viewAccount(tempUserAccountId);
+                return tempUserAccountId;
+            } catch (_) {
+                // Something went wrong! Recreate user by continuing the flow
```
Comment (vgrichina) [inline nearlib/dev.js]: I think we should log error and this message to console, to avoid confusing when debugging

File: nearlib/dev.js [hunk 2]
```diff
@@ -9,16 +9,23 @@ module.exports = {
     connect: async function(nodeUrl) {
         const studioConfig = await this.getConfig();
         const near = nearlib.Near.createDefaultConfig(nodeUrl || studioConfig.nodeUrl);
-        await this.getOrCreateDevUser();
+        await this.getOrCreateDevUser(near);
         return near;
-    }, 
-    getOrCreateDevUser: async function () {
+    },
+    getOrCreateDevUser: async function (near) {
```
Comment (vgrichina) [inline nearlib/dev.js]: I think we should either move this method to `near.dev` or keep default dev instance of `near` in `nearlib.dev`. WDYT?

Comment (janedegtiareva) [reply nearlib/dev.js]: yeah this method makes more sense in dev

Comment (janedegtiareva) [reply nearlib/dev.js]: wait it's already in dev, isn't it?

Comment (vgrichina) [reply nearlib/dev.js]: it's in `nearlib.dev`. `nearlib.dev` doesn't have connection to node (so you currently pass `near` as param).

File: nearlib/dev.js (hunk 1 above)
Comment (janedegtiareva) [reply nearlib/dev.js]: done

File: nearlib/dev.js (hunk 2 above)
Comment (janedegtiareva) [reply nearlib/dev.js]: This is finicky to change/test. I prefer to do it in next iteration to avoid last minute bug fixes tomorrow

**Pull request merged**